addons/bus/static/src/js/web_client_bus.js

```diff
@@ -0,0 +1,91 @@
+odoo.define('bus.WebClient', function (require) {
+"use strict";
+
+var core = require('web.core');
+var WebClient = require('web.WebClient');
+
+var _t = core._t;
+
+WebClient.include({
+    /**
+     * Detects the presence of assets in DOM's HEAD
+     *
+     * @override
+     */
+    start: function () {
+        var self = this;
+        this.assetsChangedNotificationId = null;
+        this._assets = {};
+
+        return this._super.apply(this, arguments).then(function () {
+            var $assets = $('head script[data-asset-xmlid], link[data-asset-xmlid]');
```

Comment:
assets may be anywhere (not necessary in `<head>`)
```suggestion
            var $assets = $('script[data-asset-xmlid],link[data-asset-xmlid]');
```

Maybe get any tag with this attribute
```suggestion
            var $assets = $('*[data-asset-xmlid]');
```